.github/workflows/build-test.yml

```
@@ -90,15 +90,22 @@ jobs:
       - run: ./tools/ci/build-test-ubuntu-with-sanitizers.sh gcc
 
   build-test-ubuntu-with-sanitizers-clang-latest:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

We can use Ubuntu latest here to reduce maintenance effort
```suggestion
    runs-on: ubuntu-latest
```